### PR TITLE
Introduction to PHP markdown addition

### PR DESCRIPTION
**Pull request by SpaceJesusTM**
Adding a introduction to PHP section to the Tech Stacks part of the repo for my contribution to A2.

### CODE REVIEW

File: Topics/Tech_Stacks/PHP.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+## Introduction
+PHP stands for Hypertext Preprocessor, and it is a server-side scripting language designed for web development. This makes it different from client-side languages such as JavaScript, which is executed on the client's browser rather than the web server. PHP is embedded within HTML code and is widely used to create dynamic and interactive web pages. It can be used to perform various tasks such as collecting form data, generating dynamic page content, managing databases, handling cookies and sessions, and more.
+
+Before continuing to learn PHP it is recommended to have a basic understanding of the commmon web development languages:
+- HTML
+- CSS
+- JavaScript
+
+## Why use PHP
+You may have heard PHP reffered to as a "dying language", but it is still widely used in web development. In fact, it is estimated that over 75% of websites use PHP in some way. PHP is a great choice for web development because it is easy to learn and use, as well as being free and open-source.
```
[the remark this answers is not in the document]
Comment (SpaceJesusTM) [reply Topics/Tech_Stacks/PHP.md]: Good catch I had forgotten about that. Added the references for this section. Should be better documented now.

**Comment by SpaceJesusTM** [comment]
Ok I want to add a bit more to the Use Cases section, I was thinking maybe showing stuff about PHP forms (user input) and embedding SQL (MySQL) would be 2 good examples to have. Was just curious to know if you have any other ideas.

**Comment by Nicknames43** [comment]
> Ok I want to add a bit more to the Use Cases section, I was thinking maybe showing stuff about PHP forms (user input) and embedding SQL (MySQL) would be 2 good examples to have. Was just curious to know if you have any other ideas.

Okay sounds good. I think you should also add some JavaScript since you mention it but don't show any examples of it.

File: Topics/Tech_Stacks/PHP.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Setup
+To set up PHP on your machine, you will need:
+- A web server
+- A database
+- PHP
+
+The official PHP website provides a [guide](https://www.php.net/manual/en/install.php) on how to install PHP on your machine. Additionaly, you can find a video tutorial on how to install PHP on Windows [here](https://www.youtube.com/watch?v=DTy57Hg1jCk).
+
+## PHP Basics
```
[the remark this answers is not in the document]
Comment (Nicknames43) [reply Topics/Tech_Stacks/PHP.md:39]: These changes look good. Makes your case much better than before 👍

**Review by Nicknames43 — APPROVED** [review]
Looks good! I can't think of any additional changes for you to make.

Overall, I liked how you justified the use of PHP in place of using HTML, CSS and JavaScript separately.  I think you are good to merge.

Good Work!

**Pull request merged**